OptimizelySDK/Optimizely.cs

```diff
@@ -724,6 +725,16 @@ public List<string> GetEnabledFeatures(string userId, UserAttributes userAttribu
             return enabledFeaturesList;
         }
 
+        public OptimizelyConfig GetOptimizelyConfig()
+        {
+            var config = ProjectConfigManager?.GetConfig();
+            if (config == null)
+            {
+                return null;
+            }
+            return new OptimizelyConfigService().GetOptimizelyConfig(config);
```

Comment:
I think we don't really need to instantiate `OptimizelyConfigService`. Since it has only public method.
Can't you define 
`public static OptimizelyConfigService Instance  = new OptimizelyConfigService()` and use method directly 
`OptimizelyConfigService.Instance.GetOptimizelyConfig()`

Reply:
Yes, I can do that (Y)